docs/example/README.md

```diff
@@ -50,7 +50,7 @@ for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; d
 ```
 ### Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
 ```bash
-osm namespace add bookstore bookbuyer bookthief bookwarehouse --enable-sidecar-injection
+osm namespace add bookstore bookbuyer bookthief bookwarehouse
```

Comment:
Wait this is required with the latest code base, I think you are still using an older binary. I merged this yesterday: https://github.com/openservicemesh/osm/pull/1638

Reply:
You might be right, I'm on `Version: v0.3.0; Commit: c91c782; Date: 2020-08-12-21:49`